gateleen-queue/src/main/java/org/swisspush/gateleen/queue/expiry/ExpiryCheckHandler.java

```diff
@@ -108,45 +127,43 @@ public static Integer getExpireAfter(MultiMap headers) {
      * @return queue-expire-after time in seconds or null if nothing is found
      */
     public static Integer getQueueExpireAfter(MultiMap headers) {
-        return getExpireValue(headers, QUEUE_EXPIRE_AFTER_HEADER);
+        try{
```

Comment:
Besides the header property name, it's the exact same implementation as in _getExpireAfter_ which leads to unwanted code duplication